.github/workflows/ci.yml

```
@@ -40,6 +40,61 @@ jobs:
       - name: Run lint
         run: npm run lint
 
+permissions:
+  contents: read
+
+jobs:
```

actionlint (1.7.12)

[error] key "jobs" is duplicated in "workflow" section. previously defined at line:6,col:1

(syntax-check)